work9/index.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+const ul = document.getElementById("js-list");
+const data = [{
+  to: "bookmark.html",
+  img: "1.png",
+  alt: "画像1",
+  text: "ブックマーク",
+},
+{
+  to: "message.html",
+  img: "2.png",
+  alt: "画像2",
+  text: "メッセージ"
+},
+]
+
+function getData() {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(data);
+
+      reject(new Error("エラー"));
+
+    }, 3000);
+  })
+}
+
+async function createElement() {
+  const fragment = document.createDocumentFragment();
+
+  const getObj = await getData();
```

Comment:
`const getObj` look like function name. change other.

Reply:
I'm sorry.

Please confirm that the correction has been completed.

f53ee5b

Reply:
promise as function name be best avoided.

Reply:
confirmed

Changed the function name


2c948c7

Reply:
Please confirm that the function name has been changed.


2c948c7

Reply:
your code is here.

```js
async function getImgArray() {
  const arrayData = await getData();
  createElement(arrayData);
}
```
`getImgArray` job is just only get images in Array.
but your code is run create Element.

Reply:
The getImgArray function means to get an array in the meaning of the word.
That means that the function name and the content being processed do not match.


94ce008